routes/index.js

```diff
@@ -105,4 +107,21 @@ const routes = [
   },
 ];
 
-module.exports = routes;
+function stripUnknownParamsOnProduction(route) {
+  const inProduction = (process.env.NODE_ENV === 'production');
+  const result = _.cloneDeep(route);
```

Comment:
There was some issue with the tests. For some reason, when invalidating the require cache and re-loading the file, we still got the old routes. I didn't enter this rabbit hole :P